app-modules/service-management/database/migrations/2023_09_05_144251_create_service_requests_table.php

```diff
@@ -45,13 +45,14 @@ public function up(): void
             $table->uuid('id')->primary();
 
             $table->string('service_request_number')->unique();
+            $table->string('title')->nullable();
             $table->string('respondent_type');
             $table->string('respondent_id');
             $table->longText('close_details')->nullable();
             $table->longText('res_details')->nullable();
 
             $table->foreignUuid('service_request_form_submission_id')->nullable()->constrained('service_request_form_submissions');
-            $table->foreignUuid('division_id')->constrained('divisions');
+            $table->foreignUuid('division_id')->nullable()->constrained('divisions');
```

Comment:
This will need to be added in a separate migration. Including a `down` method and we will need to ensure that the app will not break if the code is deployed before the migration is run.